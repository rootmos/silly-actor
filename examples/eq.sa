(system
  [(init Main ())]
  (define (Main)
    [_ (seq
         (output (= 1 1))
         (output (= 1 2))
         (output (= 1 foo))
         (output (= foo foo))
         (output (= foo bar))
         (output (= () ()))
         (output (= 1 ())))]))
