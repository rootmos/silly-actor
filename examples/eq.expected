(atom . true)
(atom . false)
(atom . false)
(atom . true)
(atom . false)
(atom . true)
(atom . false)
